Hide/Show Bugfix

wpmonty opened the pull request:
*FIXED: Data no longer overwritten if field is hidden

Commit:
Hide/Show Bugfix

*FIXED: Data no longer overwritten if field is hidden

Commit:
Added Hide Row Function

*ADDED: Hide Row function that calls setting name and echoes display
none

kjbenk commented on functions/common.php:
```diff
@@ -168,3 +167,18 @@ function gnt_force_redirect($url) {
+ * @echo string 'display:none;' if setting is disabled
+ */
+ function gnt_hide_row($setting) {
+     error_log($setting);
```
Please make sure you do not commit code with `error_log()` statements

wpmonty replied:
:)

kjbenk commented on functions/common.php:
```diff
@@ -168,3 +167,18 @@ function gnt_force_redirect($url) {
+ * @param  string $setting The setting to check
+ * @echo string 'display:none;' if setting is disabled
+ */
+ function gnt_hide_row($setting) {
```
It might be good to rename the function `gnt_should_hide_row_e` since the function echoes something and does not return it.  This is similar to the `__()` and `_e()` functions.

wpmonty replied:
makes sense

kjbenk commented on functions/common.php:
```diff
@@ -168,3 +167,18 @@ function gnt_force_redirect($url) {
+/**
+ * Hide Row if setting is disabled
+ * @param  string $setting The setting to check
+ * @echo string 'display:none;' if setting is disabled
```
If the function does not return anything then please add a `@return null` not `@echo string 'display:none;' if setting is disabled`

kjbenk commented on functions/common.php:
```diff
@@ -168,3 +167,18 @@ function gnt_force_redirect($url) {
 }
+
+/**
+ * Hide Row if setting is disabled
```
I would add a new line after this description.

wpmonty replied:
Should we do this for the other functions as well?

Commit:
Renamed gnt_hide_row

*RENAMED: hide row function to be more descriptive
*REMOVED: debug log statement

Commit:
Added New Readme

*ADDED: New readme.txt file
*TWEAKED: Plugin description

kjbenk commented:
I added this code in myself so I will be closing this PR.  Thanks @wpmonty :)
